src/test/elements/successfactors/achievements.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const suite = require('core/suite');
+const tools = require('core/tools');
+const chakram = require('chakram');
+const expect = require('chakram').expect;
+const achievementsCreatePayload = tools.requirePayload(`${__dirname}/assets/achievements-create.json`);
+const achievementsUpdatePayload = tools.requirePayload(`${__dirname}/assets/achievements-update.json`);
+
+suite.forElement('humancapital', 'achievements', { payload: achievementsCreatePayload }, (test) => {
+  const opts = {
+    churros: {
+      updatePayload : achievementsUpdatePayload
+    }
+  };
+  
+  test.withOptions(opts).should.supportCruds(chakram.put);
+  test.should.supportPagination();
+  test.withName('should support search by reviewed')
+    .withOptions({ qs: { where: 'reviewed  = \'true\'' }})
+    .withValidation((r) => {
+		expect(r).to.have.statusCode(200);
+		const validValues = r.body.filter(obj => obj.reviewed = 'true');
+		expect(validValues.length).to.equal(r.body.length);
+	}).should.return200OnGet();
+  test.withOptions({ qs: { select: 'achievementName' } })
+    .withName('should support select values')
+	.withValidation((r) => {		
+          expect(r.body[0]).to.contain.key('achievementName');      
```

Comment:
Would be great if we could beef this up a bit to say,
```suggestion
          expect(r.body[0]).to.contain.key('achievementName').but.not.contain.key('achievementId');      
```